src/payloads/session.ts

```diff
@@ -21,14 +22,35 @@ export const FIMSToken = () => {
   return mFIMSToken;
 };
 
-const generateSessionTokens = (): PublicSession => ({
+const tokensObject: PublicSession = {
   spidLevel: "https://www.spid.gov.it/SpidL2" as SpidLevel,
   walletToken: getToken("AAAAAAAAAAAAA1"),
   myPortalToken: getToken("AAAAAAAAAAAAA2"),
   bpdToken: getToken("AAAAAAAAAAAAA3"),
   zendeskToken: getToken("AAAAAAAAAAAAA4"),
   fimsToken: FIMSToken()
-});
+};
+
+const generateSessionTokens = (): PublicSession => tokensObject;
+
+export const generateCustomObjectSessionTokens = (
+  tokenString: string | ParsedQs | string[] | ParsedQs[]
+): Partial<PublicSession> => {
+  // If no tokenString is provided, return the entire tokensObject
+  if (!tokenString || typeof tokenString !== "string") {
+    return tokensObject;
```

Comment:
Shouldn't be better here to get the entire payload from `customSession` instead of `tokensObject`. Actually they are the same but if in the future we randomize the SSO Token when we randomize the session token during a fast login refresh, the variable with fresh data'd be `customSession`. Am I right? 
Just treat `tokenObject` as the default initial value. What do you think?
```suggestion
    return customSession;
```

Reply:
I applied the change, it should work now! 🚀